### ISSUE TITLE
Generated (x,y) positions are off

### ISSUE THREAD

**Issue by etpinard** [dfcreative/regl-scattermatrix#2]
@dfcreative 

From

```js
const regl = require('regl')({ extensions: 'oes_element_index_uint' })
const createMatrix = require('./')


// create splom instance
let splom = createMatrix(regl)

splom.update({
  data: [
    [1, 2, 3],
    [4, 6, 8]
  ],
  ranges: [
    [0, 10],
    [0, 10]
  ]
})

splom.draw()
```

I get:

![image](https://user-images.githubusercontent.com/6675409/37991342-e9aba62c-31d6-11e8-9cde-4d47545e373b.png)

where the the top-left panel which is supposed to plot `x=[1,2,3]` vs `y=[4,6,8]` is plotting instead `x=[4,6,8]` vs `y=[1,2,3]` and similarly for the bottom-right panel.

**Comment by dfcreative** [comment]
Whoops, sorry. Fixed 6a6cd43b514769e7ecc9a400cd75f8e2325184ea

**Comment by etpinard** [comment]
Wow that was fast. Thanks!!!
